netsim/ansible/templates/initial/ios.j2

```diff
@@ -14,12 +14,16 @@ ip host {{ hname }} {{ addr_list|join (' ') }}
 !
 {% if af.ipv4|default(False) %}
```

Comment:
```and role != 'host'`` ?

Reply:
Yeah, that was the original idea.

However, IPv4 routing on IOS is special. If you turn off "ip routing", it becomes an IPv4 host from the 1980s. It does not display the IP routing table, and probably does not use the static routes (just the default gateway). I decided leaving IPv4 forwarding enabled is the lesser evil.

Reply:
... added as a caveat